.github/workflows/python-package.yml

```
@@ -0,0 +1,36 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 2.7
+          - 3.5
+          - 3.6
+          - 3.9
+          - pypy
```

oh, they use pypy2 in GHA
```suggestion
          - pypy2
```